src/entities/candidate/candidate.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Candidate } from './candidate.entity';
+
+@Injectable()
+export class CandidateService {
+  constructor(
+    @InjectRepository(Candidate)
+    private candidateRepository: Repository<Candidate>,
+  ) {}
+
+  findAll(): Promise<Candidate[]> {
+    return this.candidateRepository.find();
+  }
+
+  findOneById(id: number): Promise<Candidate | null> {
+    return this.candidateRepository.findOneBy({ id });
```

Comment:
Maybe it's better to use
`this.candidateRepository.findOne(id);` or`this.candidateService.findOneById(id);`?

Reply:
`findOneById` is deprecated, `findOne` looks like `findOneBy` but has more options for making a query (e.g. where/select/etc options)
`...repository.findOneBy({id}) `the same as` ...repository.findOne({ where: {id} })`

I renamed all functions in services and controllers in same style `findOneBy`

Reply:
Thank you!